ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/altair/statetransition/epoch/EpochProcessorAltair.java

```diff
@@ -123,18 +123,20 @@ public void processInactivityUpdates(
 
       // Increase inactivity score of inactive validators
       final UInt64 currentScore = inactivityScores.getElement(i);
+      UInt64 newScore = currentScore;
       if (validatorStatus.isNotSlashed() && validatorStatus.isPreviousEpochTargetAttester()) {
         if (currentScore.isGreaterThan(0)) {
-          inactivityScores.setElement(i, currentScore.decrement());
+          newScore = currentScore.decrement();
         }
```

Comment:
you dont want to just use minusMinZero here?

Reply:
Probably makes sense to now that it's now updates the list straight away.  Didn't want to have to write to the list if the value hadn't changed.